tooling/cli/src/build.rs

```diff
@@ -238,13 +143,6 @@ pub fn command(mut options: Options, verbosity: u8) -> Result<()> {
     // set env vars used by the bundler
     #[cfg(target_os = "linux")]
     {
-      use crate::helpers::config::ShellAllowlistOpen;
-      if matches!(
-        config_.tauri.allowlist.shell.open,
-        ShellAllowlistOpen::Flag(true) | ShellAllowlistOpen::Validate(_)
-      ) {
-        std::env::set_var("APPIMAGE_BUNDLE_XDG_OPEN", "1");
-      }
       if config_.tauri.system_tray.is_some() {
```

Comment:
Did we move this to the shell plugin?

Reply:
not yet, we still need to figure out a safe way to handle this since the env var is set on the CLI.

Reply:
how about adding it in the plugin build script?

Reply:
Yeah but to pass a security review it would have to be specific to that env var I think

Reply:
well, the build script could read the `TAURI_CONFIG` env var, validate the scope using this same logic and then set the env var if appropriate

Reply:
the build script is running on its own process though, so it actually needs a way to export the env var to tauri instead. something like what cargo does with the cargo:something printed to stdout.

Reply:
oh TIL.

Alright, I will keep this on my TODO list so I don't forget so we can discuss it later.